flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -48,7 +48,8 @@
         <!-- Integrated lighting -->
 
         <Component ID="LIGHTING">
-            <UseTemplate Name="ASOBO_LIGHTING_Knob_Pedestal_Template">
+            <!-- FLOOD LT PED -->
+            <UseTemplate Name="FBW_Stepless_Potentiometer">
                 <NODE_ID>LIGHTING_Knob_Pedestal</NODE_ID>
                 <PART_ID>LIGHTING_Knob_Pedestal</PART_ID>
                 <ANIM_NAME>LIGHTING_Knob_Pedestal</ANIM_NAME>
```

Comment:
Since those are almost always the same, the template should set the latter two to be equivalent to `NODE_ID` by default.